src/main/java/org/ow2/proactive/service/CatalogObjectService.java

```diff
@@ -31,56 +31,62 @@
 import org.ow2.proactive.exception.EntityTooLargeException;
 import org.ow2.proactive.exception.FailedRequestException;
 import org.ow2.proactive.model.CatalogObject;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 import org.springframework.web.client.RestClientException;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.springframework.web.client.RestClientException'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AUselessImportCheck)